README.md

```diff
@@ -4,6 +4,6 @@ This is the project 1 Bingo Game for CPSC312 in UBC. It is designed to investiga
 
 
 To play the game:
-- ghci
+- cabal install --lib random,containers
 - :l PlayBingo
```

Comment:
I still have some problems in import the System.Random.
Manage to use `cabal ` to make a segway